react/Carousel.js

```diff
@@ -34,9 +33,9 @@ class Carousel extends Component {
       dots: showDots,
       arrows: showArrows,
       autoplay,
-      nextArrow: <Arrow arrowClass="vtex-carousel__arrow-right" />,
-      prevArrow: <Arrow arrowClass="vtex-carousel__arrow-left" />,
-      appendDots: dots => <Dots dots={dots} />,
+      nextArrow: <Arrow cssClass="vtex-carousel-arrow__right" />,
```

Comment:
The patter shouldn't be `vtex-carousel__arrow-right` instead of `vtex-carousel-arrow__right`?